metricbeat/module/haproxy/info/_meta/data.json

```diff
@@ -0,0 +1,59 @@
+{
+  "@timestamp": "2016-08-13T19:44:21.036Z",
+  "beat": {
+    "hostname": "beathost",
+    "name": "beathost"
+  },
+  "haproxy": {
+    "info": {
+      "compress_bps_in": 0,
```

Comment:
The event conventions should be applied here: https://www.elastic.co/guide/en/beats/libbeat/5.0/event-conventions.html